src/authorization/AllStaticReader.ts

```diff
@@ -21,12 +24,20 @@ export class AllStaticReader extends PermissionReader {
     });
   }
 
-  public async handle({ credentials }: PermissionReaderInput): Promise<PermissionSet> {
+  public async handle({ credentials, accessMap }: PermissionReaderInput): Promise<PermissionMap> {
+    const entries = mapIterable(
+      accessMap.keys(),
+      (identifier): [ResourceIdentifier, PermissionSet] => [ identifier, this.mapCredentials(credentials) ],
+    );
+
+    return new IdentifierMap(entries);
```

Comment:
Ah yes, definitely. Not sure what I was thinking there. Except perhaps that I wanted to create a new object for every instance to prevent accidents but probably shouldn't worry about that.